inc/common.inc.sh

```diff
@@ -1292,7 +1292,7 @@ function displayQuotedEnum () {
     if [ -z "$trimmed_list" ]; then
         echo
     else
-        local quoted_list="'<b>${trimmed_list// /</b>', '<b>}</b>'"
+        local quoted_list="'<b>${trimmed_list// /</b>\', \'<b>}</b>'"
```

Comment:
Okay, maybe this point needs more investigations. In my environment, the tests don't pass because the quotes disappear. For example, a `twgit feature list` prints:

`/!\ Following branches are out of process: 'origin/rm-1, origin/rm-2, origin/rm-3'!`

I am on Ubuntu 16.04 with bash version 4.3.46(1)-release (x86_64-pc-linux-gnu).